README.md

```diff
@@ -1237,6 +1253,27 @@ Other Style Guides
     }
     ```
 
+  <a name="classes--members-order"></a>
+  - [9.7](#classes--members-order) Always use this order for class members
+    1. static properties
+    1. constructor
+    1. methods
+
+    > Why? It's much nicer to have an organized convention so that people know where to look for a specific member of a class. 
+
+    ```javascript
+    class Foo {
+      // static properties
+      static bar = 'bar';
+      // constructor
```

Comment:
@dekdekbaloo I like to give some space for codes to breathe. How about adding an empty lines for each sections, static, constructor and methods?

Reply:
Okay